src/main/resources/application.yml

```diff
@@ -31,33 +31,20 @@ spring:
 
   flyway:
     enabled: true
+    url: jdbc:postgresql://${DATABASE_ENDPOINT}/${DATABASE_NAME}?sslmode=verify-full
 
   data:
     web:
       pageable:
         max-page-size: 20000
-#  h2:
-#    console:
-#      enabled: false
-
-  jpa:
-    database-platform: org.hibernate.dialect.PostgreSQL10Dialect
-    open-in-view: false
-    show-sql: false
-    generate-ddl: false
-    hibernate:
-      ddl-auto: none
-
-  datasource:
-    url: jdbc:postgresql://${DATABASE_ENDPOINT}/${DATABASE_NAME}?sslmode=verify-full
-    username: sa
-    password:
 
-    hikari:
-      pool-name: NomisVisitsMapping-CP
-      connectionTimeout: 1000
-      validationTimeout: 500
-      maximum-pool-size: 10
+  r2dbc:
+    url: r2dbc:postgresql://${DATABASE_ENDPOINT}/${DATABASE_NAME}
+    pool:
+      max-life-time: -1
```

Comment:
this is as recommended by https://github.com/r2dbc/r2dbc-pool/pull/155 to fix connections permanently being evicted from the pool